### ISSUE TITLE
Loading document 'body' from  JSONField instead of filesystem 

### ISSUE THREAD

**Issue by pauloxnet** [django/djangoproject.com#1630]
The `body` of a `Document` is loaded from a JSON file from the filesystem:
- https://github.com/django/djangoproject.com/blob/main/docs/models.py#L356

As we already store the JSON file in `metadata` field in the `Document`:
- https://github.com/django/djangoproject.com/blob/main/docs/models.py#L192

It would be better to load the `body` from the `metadata`

**Comment by jacklinke** [comment]
Yep. The field already exists & is populated, and is also the source from which the full-text search gets its information, so it makes sense that it should be the sole source of truth.

**Comment by pauloxnet** [comment]
My original goal when I added the JSONfield was only to be able to access all the information to create a full-text index. I was not so into the website code to know how the page load was working. I realized this only recently looking at the code for some improvement. At this point it would be better to upgrade the website to Django 5.1 so that we can use the GeneratedField to simplify this step

**Comment by pauloxnet** [comment]
I’ve updated the issue title and description to better clarify the intent, approach, and benefits, this could be a clear improvement for the whole docs system, and I hope someone with more time than I have right now might be interested in implementing it.
